common/common.scss

```diff
@@ -617,31 +617,29 @@ span.date-section {
 }
 
 .btn-primary {
-  border: 1px solid var(--tertiary, $tertiary);
+  border: 1px solid var(--tertiary);
 }
 
 // Composer
 
 .open .grippie {
   background: none;
   &:before {
-    border-color: var(--primary-low-mid, $primary-low-mid);
+    border-color: var(--primary-low-mid);
   }
 }
 
 .combo-box.category-chooser {
-  font-size: $font-0;
   .badge-wrapper span.badge-category {
-    color: var(--primary, $primary);
-    font-size: $font-up-1;
```

Comment:
This section fixes a size issue introduced by https://github.com/discourse/discourse/pull/14502